launch/exec_d_test.go

```diff
@@ -45,7 +45,13 @@ func testExecD(t *testing.T, when spec.G, it spec.S) {
 			h.AssertNil(t, err)
 			wd, err := os.Getwd()
 			h.AssertNil(t, err)
-			path = filepath.Join(tmpDir, "execd")
+
+			exe := ""
```

Comment:
```suggestion
			ext := ""
```

Reply:
I chose this var name for parity with https://github.com/buildpacks/lifecycle/blob/4bfb55d7d0a489fb3a3b7f3c7c083ff40fe85499/launch/launcher.go#L18. I don't really care what the var is called but I feel that it would be better for it to be consistent.

Reply:
I don't really care either, so I'm fine with be consistent with what already exists.